hooks/pre_commit/13-tuning.rb

```diff
@@ -1,7 +1,8 @@
 if app_option?(:tuning)
   # A mapping of tuning profile to the required CPU cores and memory in GB
   TUNING_SIZES = {
-    'default' => { cpu_cores: 1, memory: 8 },
+    'development' => { cpu_cores: 1, memory: 8 },
```

Comment:
https://github.com/theforeman/forklift/blob/0c965ed9a48796781b1292df222f2f0edacbd6c5/vagrant/lib/forklift/box_distributor.rb#L27 says 6 -- do you intend to adjust that?

Reply:
6 is what our nightly pipeliness pass with, so yeah, def do-able.
(even tho I would think increasing these could have benefits to the pipes…)

Reply:
> 6 is what our nightly pipeliness pass with, so yeah, def do-able. (even tho I would think increasing these could have benefits to the pipes…)

You have a suggestamendation?

Reply:
I do not, no.